x/farming/types/keys.go

```diff
@@ -27,9 +27,9 @@ const (
 
 var (
 	// param key for global farming plan IDs
-	GlobalPlanIdKey           = []byte("globalPlanId")
+	GlobalPlanIDKey           = []byte("globalPlanId")
 	GlobalLastEpochTimePrefix = []byte("globalLastEpochTime")
-	GlobalStakingIdKey        = []byte("globalStakingId")
+	GlobalLastStakingIDKey    = []byte("globalLastStakingId")
```

Comment:
`GlobalPlanIdKey ` and `GlobalStakingIdKey ` operate in the same form, but consistency is broken to attach `Last` only to `GlobalStakingIdKey `. It also needs an update on Spec to change.

Reply:
I'll revert back to the original name